tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
```

Comment:
Do you have a packaging pipeline too?

Reply:
it is going to be a separated PR. Currently I am still debugging the pipeline.